forwards_paper.tex

```diff
@@ -143,27 +148,34 @@ \section*{Introduction}
 is equivalent to having generated them during the simulation.
 Precisely, we would need to know the genealogical tree relating all sampled individuals
 at each position along the genome.
-In this paper, we describe how algorithmic tools and data structures developed for the
-coalescent simulator \msprime{}
-can be used to efficiently record, and later process, this history.
-In so doing we record the \emph{population pedigree} --
+This \emph{population pedigree} is 
```

Comment:
With your changes, it says that 'all trees' = 'population pedigree', but to get the trees you need the recombinations also.  I see you moving the "in this paper" bit to later (good idea); how about 
> Precisely, we would need to know the genealogical tree relating all sampled individuals at each position along the genome,
> which can be efficiently recorded and processed with algorithmic tools and data structures developed for the coalescent simulator \msprime{}.

Reply:
the end of this sentence says 

> including the genetic outcomes of each ancestral
meiosis.

which seems like = 'all trees'. Unless I'm missing something (likely).

Anyway the larger point of my edit is to move stuff about "efficient recording and processing with \msprime" into the part that's clearly saying what we did in this paper. So even if I'm missing something I'd like to rephrase in a way that keeps that effect

Reply:
ah, good point, that sentence should be adjusted too.  the next sentence starts "embellished pedigree", which is correct.   leave it how you like, I'll come back to this.